www/src/data/sidebars/doc-links.yaml

```diff
@@ -275,6 +275,8 @@
                   link: /docs/tailwind-css/
                 - title: Using Bulma
                   link: /docs/bulma/
+                - title: Using Theme UI
```

Comment:
> We had been thinking about it in the themes perspective, but this is probably right. And we can link to the guide from the themes section

One basic question: does Theme UI to scenarios where themes aren't involved? That might determine where it should live. But my intuition is that it's fine–it would probably be more intuitive to find it here from a styling perspective, with links from the themes docs.

Reply:
I think it does apply to other scenarios – the only part that's explicitly tied to themes is the plugin. If we'd rather put together Gatsby theme-specific docs for using Theme UI, I could try to write a doc that's more geared towards that use case.

Reply:
I think both could be useful. We could start with this and some links, and make further improvements if necessary?

Reply:
I think separate Gatsby theme-specific docs for using Theme UI would be helpful -- particularly for how using `gatsby-plugin-theme-ui` affects usage of multiple themes composed in a single project.